mitxpro/settings.py

```diff
@@ -56,6 +56,7 @@
     "mitxpro",
     "courses",
     "users",
+    "localdev.seed",
```

Comment:
Minor nitpick: What do you think of adding this to INSTALLED_APPS only if `DEBUG=True` for example, so it won't be included on production?

Reply:
Great idea. I decided to depend on the `MITXPRO_ENVIRONMENT` setting though. `DEBUG` isn't necessarily true when you're running on dev